docs/book/book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Francesco Ilario"]
```

Comment:
Maybe change this to `The Primaza Authors`

Reply:
great catch!